specs/agents/logging.md

```diff
@@ -0,0 +1,36 @@
+## Agent logging
+
+### `log_level` configuration
+
+Sets the logging level for the agent.
+
+This option is case-insensitive.
+
+|                |   |
+|----------------|---|
+| Valid options  | `trace`, `debug`, `info`, `warning`, `error`, `critical`, `off` |
```

Comment:
does `critical` translate to `fatal`, can we add an explanation under mapping section?

Reply:
`critical` can translate to `fatal` for agents which have that as an option. The mapping will be up to the agents in question.